.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags: [ 'v*.*.*' ]
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Extract release changelog
+        run: |
+          version=${GITHUB_REF#refs/tags/v*}
+          mkdir -p tmp
+          sed sed '/^# '$version'/,/^# /!d;//d;/^\s*$/d' CHANGELOG.md > tmp/release_changelog.md
+      - name: Release
+        uses: goreleaser/goreleaser-action@5df302e5e9e4c66310a6b6493a8865b12c555af2
+        with:
+          distribution: goreleaser
+          version: v1.1.0
```

why `v1.1.0`? I see the [latest version is v1.2.1](https://github.com/goreleaser/goreleaser/)

---

```suggestion
          version: v1.2.1
```

I just missed them! `1.2.0` and `1.2.1` were released 15 hours ago, this PR is 20 hours old. 😆 

Let's upgrade, the SBOM stuff in `v1.2.0` looks interesting!